### PR TITLE
Add Impler

### PR DESCRIPTION
**Pull request by chavda-bhavik**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [ ] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/impler.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# software name
+name: "Impler"
+# URL of the software project's homepage
+website_url: "https://impler.io"
+# URL where the full source code of the program can be downloaded
+source_code_url: "https://github.com/implerhq/impler.io"
+# description of what the software does, shorter than 250 characters, sentence case
+description: "Data import experience, built for engineering teams to help them build rich data import experience without constantly reinventing the wheel."
```
Comment (nodiscc) [inline software/impler.yml:8]: The description should explain what the software does.

File: software/impler.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+platforms:
+  - Nodejs
+  - Docker
+# list of tags (categories), see https://github.com/awesome-selfhosted/awesome-selfhosted-data/tree/master/tags for the full list of tags
+tags:
+  - Software Development - Low Code
+# (optional, yes/no, default no) whether the software depends on a third-party service outside the user's control
+depends_3rdparty: no
+# (optional) link to an interactive demo of the software
+demo_url: "https://web.impler.io/auth/signin"
```
Comment (nodiscc) [inline software/impler.yml:22]: Where can I find login credentials for the demo? If the demo requires signin, please remove it.

**Review by nodiscc — CHANGES_REQUESTED** [review]
Please also remove all comments from the file

File: software/impler.yml (hunk 1 above)
Comment (chavda-bhavik) [reply software/impler.yml]: Done!

File: software/impler.yml (hunk 2 above)
Comment (chavda-bhavik) [reply software/impler.yml]: Yes, users have to sign up to use the platform. Removed `demo_url`.

File: software/impler.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+source_code_url: "https://github.com/implerhq/impler.io"
+description: "Add CSV & Excel file import experience in products. Provides import widget for uploading data files, mapping columns, and reviewing data during import. Receive imported data via webhooks. Supports dynamic and static data validation."
+licenses:
+  - MIT
+platforms:
+  - Nodejs
+  - Docker
+tags:
+  - Software Development - Low Code
+depends_3rdparty: no
```
Comment (KieranRobson) [inline software/impler.yml:12]: As the default is already no, this could also be removed.

**Review by KieranRobson — CHANGES_REQUESTED** [review]
One more quick change.

**Review by KieranRobson — APPROVED** [review]

**Comment by nodiscc** [comment]
I don't think it is it possible to build a fully functional application using just Impler - it is meant to be integrated in other applications to provide data import facilities. We don't list libraries/SDKs as this would significantly increase the scope of the list. Other solutions listed on https://awesome-selfhosted.net/tags/software-development---low-code.html are web interfaces that can be used to build full end-user applications, which is not the case here.

**Comment by chavda-bhavik** [comment]
Hey @nodiscc I agree with your point, that `Impler` does not fit properly into the `Low Code` category. But `Impler` is more than just a library or SDK. 

It's a readymade infrastructure that dev teams can host by themselves, and it reduces their 80% of efforts on building it by themselves which takes weeks and months sometimes.

It's similar to the `Communication Video Conferencing` solution [livekit](https://awesome-selfhosted.net/tags/communication---video-conferencing.html#livekit).

I would be happy to know your views.

**Comment by nodiscc** [comment]
After reviewing this again, I think that livekit should be removed. It was overlooked when it was first submitted (https://github.com/awesome-selfhosted/awesome-selfhosted/pull/2628)

**Comment by chavda-bhavik** [comment]
I can understand your concern @nodiscc thank you so much for looking into PR. Wishing you the best day ahead.

**Pull request closed without merging**